packages/react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -252,7 +251,7 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
         getGroupItemLimit={groupProps && groupProps.getGroupItemLimit}
         group={group}
         groupIndex={groupIndex}
-        groupNestingDepth={groupNestingDepth}
+        groupNestingDepth={1}
```

Comment:
For sub-groups this value comes from `group.level'.  So strange that for the first level of grouping it did amount to the deepest group level ( or what should have been given that the algorithm was incorrect ).